Documentation/MigrationGuide.md

```diff
@@ -15,7 +29,7 @@ You should replace invocations in your config files with the new `ib` command na
 
 We've (finally) removed the bundled `swift2` templates. These have been deprecated for a while and were not being tested, so we're dropping them with this release. We assume most of you are already developing in a newer version of Swift (3 or 4).
 
-A few minor template functionality changes have been made, mostly cleaning up some old code and old behaviours. The `xcassets` templates now support `NSDataAssets`, and they will now only group assets if the "Provides Namespace" checkbox is ticked for a group (you can change back to the old behaviour if needed). You can read more about it in the [templates Migration Guide](Templates/MigrationGuide.md#swiftgen-60-migration-guide).
+A few minor template functionality changes have been made, mostly cleaning up some old code and old behaviours. The `xcassets` templates now support `NSDataAssets`, and they will now only group assets if the "Provides Namespace" checkbox is ticked for a group (you can change back to the old behaviour if needed). You can read more about it in the [templates Migration Guide](templates/MigrationGuide.md#swiftgen-60-migration-guide).
```

Comment:
Just reading this one last time: Maybe the thing about `NSDataAssets` + provides namespace would make more sense in a dedicated `###` paragraph?

Reply:
And in fact same for the drop of swift2 templates.

Feels strange otherwise that the `##` paragraph has a long intro listing stuff inside a text, then a `###` sub-section only for the split of storyboards… Better uniformize then?